accounts/models.py

```diff
@@ -1,3 +1,29 @@
 from django.db import models
 
-# Create your models here.
+from common.models import TimeStampedModel
+from users.models import Users
+
+
+class Account(TimeStampedModel):
+    number = models.CharField(max_length=16, unique=True)
+    is_debit = models.BooleanField(default=True)
+    CVV = models.IntegerField()
+    pincode = models.IntegerField()
+    expiration_date = models.DateField()
+    amount = models.FloatField()
+    credit_limit = models.FloatField(null=True)
+    deposit_amount = models.FloatField(null=True)
+    credit_percentage = models.FloatField()
+    deposit_percentage = models.FloatField()
+    credit_start_date = models.DateTimeField()
+    credit_end_date = models.DateTimeField()
+    deposit_start_date = models.DateTimeField()
+    deposit_end_date = models.DateTimeField()
```

Comment:
add null=True constraint to this and 5 other upper fields

Reply:
> add null=True constraint to this and 5 other upper fields

Added constraints